### PR TITLE
Updating documentation and clarifying routes.

### PR DESCRIPTION
**Pull request by erezatccsf**
Updating documentation and clarifying routes.

### CODE REVIEW

File: server/routes.py [hunk 1]
```diff
@@ -6,33 +6,38 @@
...
+
+@app.route('/pyrex', methods=['GET', 'POST'])
+def pyrexPage():
+    form = AssignmentForm()
+    if form.validate_on_submit():
+        flash('Here is the code you submitted: {}'.format(
+            form.student_input.data))
+        return redirect(url_for('pyrexPage'))
+    return render_template('pyrex.html', title=__page_title__, form=form)
+
```
Comment (erezatccsf) [inline server/routes.py]: We need the index page to remain a simple status and control page. The Pyrex functionality should be a separate page, not the index page.

File: server/routes.py [hunk 2]
```diff
@@ -6,33 +6,38 @@
...
-#        '<a href="shutdown/">Shut Down Server</a>'
-#    ]
-#    return utils_html.makeHtml(__page_title__, text)
+def indexPage():
+    text = [
+        f'<b>Pyrex said, "Hello, World!" at {asctime()}.</b>',
+        f'<a href="{url_for("pyrexPage")}">Pyrex Page</a>',
+        f'<a href="{url_for("indexPage")}">Reload Page</a>',
+        f'<a href="{url_for("shutdownPage")}">Shut Down Server</a>'
+    ]
```
Comment (erezatccsf) [inline server/routes.py]: I made liberal use of the url_for() function. Very helpful.

File: server/templates/pyrex.html [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
 {% block content %}
-	<h1>Submit Yout Code</h1>
+	<h1>Submit Your Code</h1>
 	<form action="" method=post novalidate>
 		{{ form.hidden_tag() }}
 		<p>
 			{{ form.student_input.label }}<br>
-			{{ form.student_input(cols="170", rows="80") }}
+			{{ form.student_input(cols="100", rows="20") }}
```
Comment (erezatccsf) [inline server/templates/pyrex.html]: Short programs for now. No need for giant textarea. Eventually need to add scrollbars.

File: README.md [hunk 4]
```diff
@@ -180,22 +180,25 @@ But on MacOS / Linux it might look like this:
 ### Now install the additional Python modules required:
 ```
 pip install Flask==1.0.2
+pip install Flask-WTF==0.14.2
```
Comment (erezatccsf) [inline README.md]: Make sure to keep documentation updated.

**Comment by erezatccsf** [comment]
I want you to approve my changes before I merge them into your branch.

File: README.md (hunk 4 above)
Comment (rpark4) [reply README.md]: Will do.

Comment (rpark4) [reply README.md]: Will do.

**Pull request merged**